dask_planner/src/parser.rs

```diff
@@ -381,14 +394,23 @@ impl<'a> DaskParser<'a> {
                         let create = CreateTable {
                             table_schema: tbl_schema,
                             name: tbl_name,
+                            if_not_exists,
+                            or_replace,
                             with_options,
                         };
                         Ok(DaskStatement::CreateTable(Box::new(create)))
                     }
+                    _ => self.expected("'as' or 'with'", self.parser.peek_token()),
                 }
             }
             _ => {
```

Comment:
Wondering what this catchall would be catching at this point - are there other forms of `CREATE TABLE` statements not captured above, or can we just raise a parser error at this point?

Reply:
Thinking about this a bit more, do we need the `as` branch or could we just pass the catchall case back to the parser to figure things out/error as needed.